root/etc/e-smith/db/configuration/migrate/yum-cron_remove_command_prop

```diff
@@ -0,0 +1,8 @@
+{
+# We want to remove ${'yum-cron'}{'command'}
+# in later release it can be removed
+# introduced in version 0.1.7.el7
+use esmith::ConfigDB;
+my $db = esmith::ConfigDB->open;
+$db->get_prop_and_delete('yum-cron', 'command');
```

Comment:
- Always return empty string in migrate fragments
- Trim any trailing whitespace 
- The $DB variable is available and already set to ConfigDB instance